src/server/src/Server.cpp

```diff
@@ -47,17 +46,10 @@ bool Server::configure(ResourceFinder& rf)
         return false;
     }
 
-    /* Set up ports for streaming. */
-    if (!port_position_.open("/yarp-omega3-server/position:o"))
+    /* Set up port for streaming. */
+    if (!port_robot_state_.open("/yarp-omega3-server/robot_state:o"))
```

Comment:
You should also update the README.md file in order to clarify the new behavior that you introduced, i.e. all the state streamed over a single port named `/yarp-omega3-server/robot_state:o`.